### ISSUE TITLE
Use two different folders to separate the apps and packages

### ISSUE THREAD

**Issue by Neosoulink** [ever-co/ever-demand#1459]
## Suggestion for structure improvement

It will really be intersecting to move the current applications in the ``packages`` folder to a dedicated folder (``apps`` for example).

The mono-repo will devoid more conventional and practical to use, to separate the two logics

Something like that:
```
...
├── apps // Contains the applications of the Platform (which can use packages as dependencies)
|   ...
│   ├── shop-mobile-ionic
│   └── shop-web-angular
├── packages // Contains the packages of the mono-repo (which can be used as dependencies)
|   ... 
│   ├── common
│   └── core
...
```

**Comment by evereq** [comment]
Yes, so we have already the same structure in the Gauzy - https://github.com/ever-co/ever-gauzy and it's really cool. 
Last time it looks like @vercel/turborepo got a lot of traction as it's very fast and it seems also to support well such structures - https://github.com/vercel/turborepo/tree/main/examples/kitchen-sink (for Gauzy we use Nx with Lerna at the moment)

**Comment by Neosoulink** [comment]
**Amazing 🤩!**

It probably will require some modifications (and some cup of coffee). 
But it seems to me to be a good option to use, to change the project architecture and may also migrate to @vercel/turborepo 

It may take time, but it will be a big boost ⚡
